src/misc_utils.jl

```diff
@@ -1,34 +1,34 @@
-immutable DiffCache{T, S}
+immutable DiffEqDiffTools.DiffCache{T, S}
```

Comment:
Delete this entire file. It's no longer needed since all of these functions are in DiffEqDiffTools